### PR TITLE
GROOVY-8226: JSR308 initial plumbing tweaks

### PR DESCRIPTION
**Pull request by paulk-asert**

### CODE REVIEW

File: src/main/org/codehaus/groovy/vmplugin/vm8/Java8.java [hunk 1]
```diff
@@ -51,4 +53,13 @@ public int getVersion() {
         return 8;
     }
 
+    protected int getElementCode(ElementType value) {
```
Comment (jwagenleitner) [inline src/main/org/codehaus/groovy/vmplugin/vm8/Java8.java]: might be good to have `@Override`

Comment (paulk-asert) [inline src/main/org/codehaus/groovy/vmplugin/vm8/Java8.java]: done

**Comment by paulk-asert** [comment]
Actually, I am going to refine the Jira issues a little further. The current proposal doesn't push the relevant annotations to all the expected places for all cases that currently pass the grammar. This change may stand once I see what else needs to be done for the extra issues that I'll create or some aspects may move into or out of this issue.

**Comment by danielsun1106** [comment]
+1

**Pull request closed without merging**